code/game/mecha/mecha.dm

```diff
@@ -123,6 +127,7 @@
 	var/datum/action/innate/mecha/mech_zoom/zoom_action = new
 	var/datum/action/innate/mecha/mech_toggle_phasing/phasing_action = new
 	var/datum/action/innate/mecha/mech_switch_damtype/switch_damtype_action = new
+	var/datum/action/innate/mecha/mech_toggle_floorbuffer/floorbuffer_action = new
```

Comment:
put this action on /obj/mecha/janitor/nkarrdem instead please, we dont need it for every mech